Dockerfile

```diff
@@ -1,13 +1,22 @@
-FROM maven:3.3-jdk-7
+FROM jimschubert/8-jdk-alpine-mvn:1.0
```

Comment:
@fehguy At the time I wrote this, there weren't any minimal containers created for standard base images that just needed Java and Maven. Smaller base images such as the one I've created not only reduce bloat in the container, but also reduce security risks.

For example, here's what [`maven:3.3-jdk-7`](https://microbadger.com/images/maven:3.3-jdk-7) does:
* derives from `openjdk:7-jdk`
  - which installs multiple zip utilities (bzip2 unzip xz-utils) 
  - `openjdk:7-jdk` derives from `buildpack-deps:jessie-scm`
    - which installs (bzr, git, mercurial, openssh-client, subversion, procps)
       - `buildpack-deps:jessie-scm` derives from `buildpack-deps:jessie-curl`
          - which installs (ca-certificates, curl, wget)
             - `buildpack-deps:jessie-curl` derives from `debian:jessie`
                - `debian:jessie` is a base image.

Compare to what [`jimschubert/8-jdk-alpine-mvn:1.0`](https://microbadger.com/images/jimschubert/8-jdk-alpine-mvn:1.0) does:
* installs ca-certificates openssl
* curls and installs maven manually
* derives from `openjdk:8-jdk-alpine`
  - which install java
  - `openjdk:8-jdk-alpine` derives from `alpine:3.4`
     - `alpine:3.4` is a base image.

There are other points to `maven:3.3-jdk-7`, which I'll email you directly.

That said, this could just as easily be updated with the contents of the `jimschubert/8-jdk-alpine-mvn:1.0` Dockerfile.